core/dateUtils.js

```diff
@@ -118,6 +118,9 @@ export const convertDate = ({ dateStr, formatFrom = formats.dateISO, formatTo })
     return null
   }
   const dateParsed = parse(dateStr, formatFrom)
+  if (!dateParsed || Number.isNaN(dateParsed.getTime())) {
+    return null
+  }
```

Comment:
Maybe there is a better way to do this

Reply:
yes, there is, using the function isValid from date-fns for example